### DISCUSSION TITLE
Endre på CopyToClipboard

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1891] [Komponenter]
🔗 [Link til CopyToClipboard i Aksel](https://aksel.nav.no/komponenter/interne/copytoclipboard)

Vi ønsker å vurdere hva som kan forbedre brukeropplevelsen av CopyToClipboard for utviklere og designere. Har du innspill til noe som vil gjøre denne komponenten bedre og mer brukervennlig for deg og brukerne av deres løsninger?

**Comment by KenAJoh** [comment]
https://aksel.nav.no/komponenter/core/copybutton
